tests/path_gateway_tar_test.go

```diff
@@ -73,6 +76,7 @@ func TestTar(t *testing.T) {
 		},
 		{
 			Name: "GET TAR with explicit ?filename= succeeds with modified Content-Disposition header",
+			Spec: "specs.ipfs.tech/http-gateways/path-gateway/#content-disposition-response-header",
```

Comment:
@laurentsenta thoughts on having `Specs` version which allows for more than 1 URL?  I imagine there will be cases where more than one spec is relevant.

Reply:
Thought about it, you can use Specs! https://github.com/ipfs/gateway-conformance/pull/159/files#diff-73ca403802a777f0a38d9aed0daa6f62819960272f8997e91c7eb6bd6782935fR17